### DISCUSSION TITLE
How to get SNMP logs in kibana! is polling possible in security onion 2 ?

### DISCUSSION THREAD

**Discussion by rijafida** [Security-Onion-Solutions/securityonion#4352] [Unsupported Versions]
Hi,

I am a newb to security Onion and have been going through the process of self learning. I am stuck on how to get snmp logs in kibana. so far i am not receiving anything!. I have read multiple discussions on the snmp plugin and all say that the plugin is already present and the MIBs are already imported, but I do not see any setting in advanced setting that has anything to do with snmp. In version 16.04 there was a setting that allowed for different versions of snmp to be enabled.
My question is simply where to get started any clue to pointing me in the right direction !
Any help would be highly appreciated.

**Comment by weslambert** [comment]
Zeek already writes SNMP logs for SNMP traffic it witnesses traverse the network.  If you are referring to bringing SNMP logs from an external source, Logstash provides an input plugin for this:

https://www.elastic.co/guide/en/logstash/current/plugins-inputs-snmp.html
